internal/resources/fetcher.go

```diff
@@ -2,18 +2,22 @@ package resources
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"net/url"
 
+	"github.com/kubewarden/audit-scanner/internal/constants"
+	auditscannererrors "github.com/kubewarden/audit-scanner/internal/errors"
 	policiesv1 "github.com/kubewarden/kubewarden-controller/pkg/apis/policies/v1"
 	"github.com/rs/zerolog/log"
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Reminder: if we get rid of the `errors.go` file from this codebase, this aliasing is not required

Reply:
Not really. I'm using the `errors.Is` function in the code. Thus, we still need the alias.